rules/windows/defense_evasion_unusual_ads_file_creation.toml

```
@@ -0,0 +1,79 @@
+[metadata]
+creation_date = "2021/01/21"
+maturity = "production"
+updated_date = "2021/01/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious creation of Alternate Data Streams on highly targeted files. This is uncommon for legitimate files
+and sometimes done by adversaries to hide malware.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License"
+name = "Unusual File Creation - Alternate Data Stream"
+risk_score = 47
+rule_id = "71bccb61-e19b-452f-b104-79a60e546a95"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+type = "eql"
+
+query = '''
+file where event.type : "creation" and
+  file.name : "*:*" and
```

what if we changed this to the following, to eliminate the leading wildcard

```suggestion
  file.path : "C:\\*:*" and
```

---

that's a good suggestion, and tested successfully, will add